packages/sdks/src/components/blocks/blocks-wrapper.lite.tsx

```diff
@@ -132,27 +133,29 @@ export default function BlocksWrapper(props: BlocksWrapperProps) {
         reactNative: { dataSet: { class: state.className } },
         default: {},
       })}
-      style={props.styleProp}
-      // eslint-disable-next-line @builder.io/mitosis/css-no-vars
-      css={useTarget({
-        // react native's ScrollView can't accept `alignItems` in styles.
-        reactNative: {
-          display: 'flex',
-          flexDirection: 'column',
-        },
-        default: {
-          display: 'flex',
-          flexDirection: 'column',
-          alignItems: 'stretch',
-        },
-      })}
+      style={{
+        ...useTarget({
+          // react native's ScrollView can't accept `alignItems` in styles.
+          reactNative: {
+            display: 'flex',
+            flexDirection: 'column',
+          },
+          default: {
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'stretch',
+          },
+        }),
+        ...props.styleProp,
+      }}
```

Comment:
Mitosis generation of React component creates a <style> block that gets outputted outside of the block/parent div - either we move to style else we can still use CSS and specifically for react delete any subsequent <style> block

Reply:
we will not be able to delete any next element sibling when running the script, so had to write a plugin to update the placement of the style element. the inling of styles didn't feel right. let me know your thoughts. i think a better approach would be to add a useMetadata option for react - injectStyleAtTop that injects style elements at the top of the fragment

Reply:
@samijaber